src/translations/IntlContext.tsx

```diff
@@ -5,16 +5,51 @@ import en from '../compiled-lang/en.json';
 
 const Context = React.createContext(null);
 
-const IntlConsumer = (props) => {
-  const loadLocaleData = locale => {
-    switch (locale) {
-      case 'sk':
-      case 'sk-SK':
-        return sk;
-      default:
-        return en;
+//TODO waiting for what the array will look like
+const languageMessageData = [
+  {
+    "locale": "en",
+    "message": en
+  },
+  {
+    "locale": "sk",
+    "message": sk
+  }
+]
+
+//TODO I'm waiting for what the array will look like. Then refactoring the code
+const loadLocaleData = (locale) => {
+  for (let i = 0; i < languageMessageData.length; i++){
```

Comment:
Tie for cykly sa este zmenia a zrefaktoruju, podla toho z akeho array budeme vychadzat. Resp. cakam aj na to, ci je dobra logika

Reply:
sprav prosim filter a default nemaj takto hardcoded